.github/workflows/1_2_b_bump_extension_only.yml

```diff
@@ -0,0 +1,79 @@
+name: Bump versions for extension only
+
+on:
+    push:
+      branches: 
+        - 'master'
+        - '**.x'
+    workflow_dispatch: # promote patch to stable
+      branches:
+        - '**.x'
+
+env:
+  ENVIRONMENT: ${{ secrets.ENVIRONMENT }}
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v2
+      - name: Extract branch name
+        shell: bash
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+        id: extract_branch
+      - name: Install Dependencies
+        run: npm install
+      - name: checkout master if necessary
+        if: endsWith(github.ref, '/master') == false
+        run: git checkout master
+      - name: Setup Git
```

Comment:
It can happen right that both this workflow is called (by a push to dev) and `2_bump_versions.yml` is called because Prisma CLI dev was released. In that case two parallel workflows would be writing to master. Have you already considered that case?

Reply:
I was thinking of including turnstile for this and to pull any changes before starting any further steps in the workflow. This is also the case if there is a new Prisma CLI dev and Prisma CLI patch dev at the same time, since both will update the versions in the master branch as well.